stable/docker-registry/templates/deployment.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
+apiVersion: {{ template "deployment.apiVersion" . }}
```

Comment:
Why?
apps/v1 was introduced in 1.10 I believe.

Reply:
I am not questioning the api version just your implementation of it in the template.
Hiding the version we are using inside _helpers.tpl doesn't seem like a good plan to me.

Reply:
#17454 is a better implementation.